manager/src/main/java/com/redhat/service/smartevents/manager/connectors/ConnectorsApiClientImpl.java

```diff
@@ -67,15 +67,18 @@ public Connector getConnector(String connectorExternalId) {
         ConnectorsApi connectorsAPI = createConnectorsAPI();
 
         try {
-            if (LOGGER.isDebugEnabled()) {
-                LOGGER.debug(String.format("Retrieving Connector with ID '%s'", connectorExternalId));
-            }
+            LOGGER.debug("Retrieving Connector with ID '{}'", connectorExternalId);
             return connectorsAPI.getConnector(connectorExternalId);
         } catch (ApiException e) {
-            if (e.getCode() == HttpStatus.SC_NOT_FOUND) {
-                return null;
+            switch (e.getCode()) {
+                case HttpStatus.SC_NOT_FOUND:
+                case HttpStatus.SC_GONE:
+                    LOGGER.info("Connector could not be found in MC Fleet Manager.");
```

Comment:
@manstis Can we make the log statements a bit more helpful e.g 

```
LOGGER.info("Connector with id '{}' could not be found in Connector Namespace '{}'.", connectorExternalId, mcNamespaceId);
```
That should make it much easier for us to debug problems in the future.

Reply:
Done.